libs/SalesforceSDK/src/com/salesforce/androidsdk/security/BiometricAuthenticationManager.kt

```diff
@@ -42,7 +43,7 @@ internal class BiometricAuthenticationManager: AppLockManager(
     @Suppress("INAPPLICABLE_JVM_NAME")
     @get:JvmName("isEnabled")
     override val enabled: Boolean
-        get() { return currentUser != null && getPolicy(currentUser!!).first }
+        get() = currentUser?.let { getPolicy(it).first } == true
```

Comment:
This is the fix.  Since the crash rate is 0.03%, I'm thinking that lines up with `currentUser` being nulled after the the  first check but before `getPolicy`.

Reply:
I also didn't see any other instances if `!!` in our codebase.

Reply:
Nice!